src/setup_payload/tests/TestQRCode.cpp

```diff
@@ -267,13 +279,13 @@ void TestQRCodeToPayloadGeneration(nlTestSuite * inSuite, void * inContext)
     SetupPayload payload = GetDefaultPayload();
```

Comment:
Could we have such a test case but with SetupPayload set to all 1 bits? To verify that no number is truncated during the process?

Reply:
The truncation test was already added in: https://github.com/project-chip/connectedhomeip/pull/6132/files#diff-0ad3d0487a75f250c389ea1a7c571bad80e6b23ffdfe2e9082237450c9db15e6R123